### PR TITLE
Elasticsearch 2.4 and 5.2

### PR DESCRIPTION
**Pull request by Crell**
And docs therein.

### CODE REVIEW

File: src/configuration/services/elasticsearch.md [hunk 1]
```diff
@@ -57,3 +58,37 @@ if (isset($_ENV['PLATFORM_RELATIONSHIPS'])) {
...
+    type: "elasticsearch:2.4"
+    disk: 1024
+    configuration:
+        extensions:
+            - analysis-icu
+            - lang-python
+            
+```
+
+In this example you'd have the ICU analysis plugin and
```
Comment (mrkschan) [inline src/configuration/services/elasticsearch.md]: and Python language plugin.

Comment (Crell) [inline src/configuration/services/elasticsearch.md]: Duh, Crell... Fixed.

File: src/configuration/services/elasticsearch.md [hunk 2]
```diff
@@ -57,3 +58,37 @@ if (isset($_ENV['PLATFORM_RELATIONSHIPS'])) {
 }
 ```
 
+## Plugins
+
+The Elasticsearch 2.4 and later containers offer a number of plugins.  To enable them, list them under the `configuration.extensions` key in your `services.yaml` file, like so:
```
Comment (ralt) [inline src/configuration/services/elasticsearch.md]: s/containers/services? Not sure what we use elsewhere.

File: src/configuration/services/elasticsearch.md [hunk 3]
```diff
@@ -57,3 +58,37 @@ if (isset($_ENV['PLATFORM_RELATIONSHIPS'])) {
...
+```
+
+In this example you'd have the ICU analysis plugin and Python script support plugin.
+
+### Available plugins
+
+The following official Elasticsearch plugins are supported.
+
+* **analysis-icu** - Support ICU Unicode text analysis
+* **analysis-kuromoji** - Japanese langauge support
```
Comment (ralt) [inline src/configuration/services/elasticsearch.md]: language

File: src/configuration/services/elasticsearch.md [hunk 4]
```diff
@@ -57,3 +58,37 @@ if (isset($_ENV['PLATFORM_RELATIONSHIPS'])) {
...
+* **analysis-phonetic** - Phonetic analysis
+* **analysis-smartcn** - Smart Chinese Analysis Plugins
+* **analysis-stempel** - Stempel Polish Analysis Plugin
+* **cloud-aws** - AWS Cloud plugin, allows storing indices on AWS S3
+* **cloud-azure** - Micorosoft Azure plugin, allows storing indices on Azure
+* **cloud-gce** - Google Cloud Engine plugin, allows storing indices on GCE
+* **delete-by-query** - Support for deleting documents matching a givenquery
```
Comment (ralt) [inline src/configuration/services/elasticsearch.md]: given query

File: src/configuration/services/elasticsearch.md [hunk 5]
```diff
@@ -57,3 +58,37 @@ if (isset($_ENV['PLATFORM_RELATIONSHIPS'])) {
...
+## Plugins
+
+The Elasticsearch 2.4 and later services offer a number of plugins.  To enable them, list them under the `configuration.extensions` key in your `services.yaml` file, like so:
+
+```yaml
+mysearch:
+    type: "elasticsearch:2.4"
+    disk: 1024
+    configuration:
+        extensions:
```
Comment (sruggier) [inline src/configuration/services/elasticsearch.md]: This should be `plugins`, not `extensions` :)

**Pull request merged**